script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -29,10 +29,10 @@ Along with a nice meal, I'd like to spend some time with you as well."
 <Page11>
 "――――――"
 With a smile on her face, she pulls my sleeve with a curl of her fingers.
-It is really just a small gesture, but it is filled with a mysterious coercion that I could not resist.
+It is really just a small gesture, but one filled with a mysterious coercion that I could not resist.
```

Comment:
It is really just a small gesture, but one possessing a mysterious coercion that I could not resist.

Reply:
Filled is a pretty faithful tl here, since it doesn't read off to me I might be inclined to leave it
> 不思議な強制力に満ちていた。
> 満ちる (みちる) (v1,vi) (1) to be full; (v1,vi) (2) to wax (e.g. moon); (v1,vi) (3) to rise (e.g. tide); (v1,vi) (4) to mature; to expire; (P);